### PR TITLE
Feature/hosts active

### PR DESCRIPTION
**Pull request by vinay-ebi**
Display active and inactive  badge in autocomplete field based on hosts  availablity

### CODE REVIEW

**Review by radome — CHANGES_REQUESTED** [review]
Please add a migration module to `ensembl_dbcopy/migrations` in order to add the new `host.active` field.

File: ensembl_dbcopy/migrations/0004_auto_20210129_1245.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.13 on 2021-01-29 12:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```
Comment (marcoooo) [inline ensembl_dbcopy/migrations/0004_auto_20210129_1245.py:7]: I thought the PR was  only for the new field "active" I see some updates on `group` field. Is this intended? or a mix between two concurrent devs?

File: ensembl_dbcopy/models.py [hunk 2]
```diff
@@ -159,7 +160,7 @@ class Meta:
     mysql_user = models.CharField(max_length=64)
     virtual_machine = models.CharField(max_length=255, blank=True, null=True)
     mysqld_file_owner = models.CharField(max_length=128, null=True, blank=True)
-
+    active = models.BooleanField(default=True, blank=False)
```
Comment (marcoooo) [inline ensembl_dbcopy/models.py:163]:
```suggestion
    active = models.BooleanField(default=True, blank=False)
    
```

File: ensembl_dbcopy/static/js/multiselect.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ function hostStringToDetails(string) {
 }
 
 function hostDetailsToString(details) {
-  return details.name + ":" + details.port;
+  return details.name + ":" + details.port ;
```
Comment (marcoooo) [inline ensembl_dbcopy/static/js/multiselect.js:50]: If `hostDetailsToString` is not used anymore, please remove it.

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: ensembl_dbcopy/migrations/0004_auto_20210129_1245.py (hunk 1 above)
Comment (vinay-ebi) [reply ensembl_dbcopy/migrations/0004_auto_20210129_1245.py:7]: I did the migration while working for groups so it picked those changes . i will redo migrations

File: ensembl_dbcopy/static/js/multiselect.js (hunk 3 above)
Comment (vinay-ebi) [reply ensembl_dbcopy/static/js/multiselect.js:50]: hostDetailsToString is used better to keep it

File: ensembl_dbcopy/migrations/0004_auto_20210129_1245.py (hunk 1 above)
Comment (vinay-ebi) [reply ensembl_dbcopy/migrations/0004_auto_20210129_1245.py:7]: I recreated the migration file and i see the all the changes which are not related to changes i made .

I think these are previous migrations not applied  : 

https://github.com/Ensembl/ensembl-production-services/pull/115/files#diff-45432eec4270f212866cd5050af5fb0b139ab76887c4989eddac88b0f4ff31eeR25 

https://github.com/Ensembl/ensembl-production-services/pull/115/files#diff-45432eec4270f212866cd5050af5fb0b139ab76887c4989eddac88b0f4ff31eeR30

**Review by marcoooo — DISMISSED** [review]

**Comment by vinay-ebi** [comment]
> Please add a migration module to `ensembl_dbcopy/migrations` in order to add the new `host.active` field.

migration script added

**Review by radome — APPROVED** [review]

**Pull request merged**